### DISCUSSION TITLE
Launched a Plenti site for the Australian SAS Historical Foundation

### DISCUSSION THREAD

**Discussion by jamestagal** [plentico/plenti#346] [Show and tell]
hi all,

Just wanted to show off the latest site I built using Plenti that I launched today. I've pretty happy with the results and it was a huge project for me as I only do this part time at present. Special thanks to @jimafisk for all the support, advice along the way. 🙏 
Please check it out and let me know what you think. 

https://australiansas.com/

Overview of the site:
- Plenti CMS with custom templates from CodeStitch
- 22 pages including the index linked from the nav
- 43 archive news posts listing pages
- 11 history pages 
- 1 blog content type for all posts (including archive news posts and current)
- appox 90 unique Svelte components
- lots of custom field widgets for adding content to mainly the Memorial and Citation pages
- custom image components for news posts
- tonnes of images optimised for the web using the Sharp image processing library

Ben

**Comment by jimafisk** [comment]
Wow @jamestagal, the site looks really awesome :heart_eyes:. Great work, you did an amazing job with this!! Huge congrats on the launch :tada:

**Reply by jamestagal** [reply]
Thanks @jimafisk 

I also see it's getting lots of positive page speed results:

Cloudflare:
<img width="1140" alt="Screen Shot 2024-10-24 at 5 14 16 pm" src="https://github.com/user-attachments/assets/cdfd8609-72a6-4706-996d-6917e0794538">

Google page speed:

<img width="983" alt="Screen Shot 2024-10-24 at 5 13 54 pm" src="https://github.com/user-attachments/assets/87392104-6f42-4b9e-b6dd-335f116c0401">

**Reply by jimafisk** [reply]
Those are some good scores @jamestagal! Really nicely done :)
